Documentation/Architecture/CrossPlatform/DeterminingPlatformCapabilities.md

```diff
@@ -0,0 +1,60 @@
+# Determining platform capabilities
+
+A common question asked of the MRTK involves knowing which specific device (ex: Microsoft HoloLens 2) is being
+used to run an application. The goal behind that knoweldge generally centers around knowing whether or not a
+particular feature (ex: articulated hands) is available.
+
+Mixed reality devices are evolving rapidly and any attempt at capturing a table mapping features to specific
+hardware will quickly become untennable.
+
+## Checking for platform capabilities
+
+To help answer the root question of whether or not an application can leverage a specific feature, the Mixed Reality
+Toolkit provides the [IMixedRealityCapabilityCheck](xref:Microsoft.MixedReality.Toolkit.IMixedRealityCapabilityCheck)
+interface.
+
+### Querying for a capability
+
+The most common pattern for determining support for a specific capability is to query the service.
+
+The following example checks to see if the input system has loaded a data provider with support for articulated hands.
+
+```
+bool supportsArticulatedHands = false;
```

Comment:
I think it would be good to show the entire code snippet to make this work - the next question is, how do I get the input system?